utils/__init__.py

```diff
@@ -1 +1 @@
-""" Bobby B Bot utilities module """
\ No newline at end of file
+""" Bot utilities module """
```

Comment:
We could probably even remove Bot, since we know that's what it's for no? So just Utilities Module? Open to leaving it if you feel strongly. Also newline :)

Reply:
I just removed Bobby B without real consideration for what was left there. I have no feelings one way or another so will remove.